R/post_data.R

```diff
@@ -16,6 +16,9 @@ post_data <- function(table, data, endpoint) {
 
   api_url <- paste0(endpoint, table)
   api_url <- file.path(dirname(api_url), basename(api_url), "")
+  if(substring(api_url,nchar(api_url)) != "/"){
+    api_url <- paste0(api_url, "/")
```

Comment:
@RyanJField - why does the api url need a `/` on the end of it? I didn't think this was required?

Reply:
The registry will respond with a 301 if it doesn’t have a slash (not sure if this is windows specific).

Reply:
Weird. Not a mac issue, but fine.